jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/AbstractCompilerMojo.java

```diff
@@ -255,6 +269,11 @@ protected JoocConfiguration createJoocConfiguration(Log log) throws MojoExecutio
     configuration.setMigrateToTypeScript(migrateToTypeScript);
     configuration.setUseEcmaParameterInitializerSemantics(useEcmaParameterInitializerSemantics);
     configuration.setSuppressCommentedActionScriptCode(suppressCommentedActionScriptCode);
+    configuration.setNpmPackageNameReplacers(
+            npmPackageNameReplacers.stream()
+                    .map(config -> new SearchAndReplace(Pattern.compile(config.getSearch()), config.getReplace()))
+            .collect(Collectors.toCollection(ArrayList::new))
```

Comment:
Isn't there simply a Collectors.toList()? Or is our max. Java version too old to use that one?